stellarphot/settings/tests/test_photometry_runner.py

```diff
@@ -0,0 +1,100 @@
+import os
+from pathlib import Path
+
+import ipywidgets as ipw
+import pytest
+from astropy.io import fits
+
+from stellarphot.settings import (
+    PhotometrySettings,
+    PhotometryWorkingDirSettings,
+    settings_files,
+)
+from stellarphot.settings.custom_widgets import PhotometryRunner
+from stellarphot.settings.tests.test_models import DEFAULT_PHOTOMETRY_SETTINGS
+
+
+# See test_settings_file.TestSavedSettings for a detailed description of what the
+# following fixture does. In brief, it patches the settings_files.PlatformDirs class
+# so that the user_data_dir method returns the temporary directory.
+@pytest.fixture(autouse=True)
+def fake_settings_dir(mocker, tmp_path):
+    mocker.patch.object(
+        settings_files.PlatformDirs, "user_data_dir", tmp_path / "stellarphot"
+    )
+
+
+class TestPhotometryRunner:
+
```

Comment:
Spurious blank line after class definition?  I don't think [PEP8 recommends this](https://peps.python.org/pep-0008/#blank-lines) (feel free to disprove me, I am only about 70% sure)... should probably remove.

Reply:
Seems to pass ruff either way so I deleted the blank line